rumqttc/src/eventloop.rs

```diff
@@ -185,7 +185,7 @@ impl EventLoop {
             o = network.read() => {
                 let incoming = o?;
                 if let Some(packet) = self.state.handle_incoming_packet(incoming)? {
-                    network.send(packet).await?;
+                    network.write(packet).await?;
```

Comment:
This packet is never flushed unless there's a ping or `options.max_request_batch - 1` more packets "sent".

Reply:
Had the same doubt, was using tests as a crutch, but I see how even the tests aren't complete.

What type of acks should we instantly respond to, I guess only `PingResp`?

Reply:
I'm not so deep in the MQTT standard but think it's just `PingResp`.